enemy/enemy_alt.gd

```diff
@@ -0,0 +1,183 @@
+class_name EnemyAlt
+extends CharacterBody2D
+
+signal hit(damage: float)
+signal died
+
+@export var hitbox: CollisionShape2D
+@export var sensor: Area2D
+@export var anim: AnimatedSprite2D
+@export var hit_particle: Resource
+@export var hit_particle_color: Color
+@export var attack_sfx: AudioStreamPlayer2D
+
+@export var health: int = 2
+@export var damage: int = 1
+@export var speed: float = 100.0
+@export var coin_reward: int = 1
```

Comment:
went with a lot of `@export`s in this approach.

although doing thing programmatically is great, its really simple to wire things up and make adjustments to the scene instead of having to make an inherited version of the script just to override a value.